### DISCUSSION TITLE
Magic Forms plugin was abandoned

### DISCUSSION THREAD

**Discussion by skydiver** [skydiver/october-plugin-forms#267] [General]
Magic Forms was supported only on OctoberCMS v1.

Since newest versions of OctoberCMS became a "paid / subscription / not open source" software i've decided to end all my contributions.

All my plugins where abandoned.
I left the GitHub repos public just in case someone want to fork, test, etc.

Thanks for your comprehension.
Martin.-

**Comment by blakej115** [comment]
There is now an actively maintained fork of this plugin for October CMS v3:
https://octobercms.com/plugin/blakejones-magicforms
https://github.com/blakej115/magic-forms

Thank you @skydiver for all of your hard work on this plugin. The October CMS community appreciates and relies heavily upon it.
